src/views/List.jsx

```diff
@@ -3,6 +3,10 @@ import { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { comparePurchaseUrgency } from '../api/firebase';
 
+import copy from 'clipboard-copy';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClipboard as clipboard } from '@fortawesome/free-regular-svg-icons';
+
 export function List({ data, listId }) {
 	const navigate = useNavigate();
 
```

Comment:
**thought:** We now have 3 components `WelcomePrompt`, `CopyToken`, `FormAndList` nested within our `List` view component. Perhaps we consider moving those into their own files within the `Component` folder for cleaner organization. OR we could at least move `WelcomePrompt` and `CopyToken` since they are smaller components, and leave `FormAndList` where it is since it's really the main focus of the `List` page. 

Not saying this needs to be implemented. Just a thought to discuss together!

Reply:
I agree moving the `WelcomePrompt` and `CopyToken` components to their own files. I'm scared to touch the `FormAndList` component. Anytime I've tried refactoring it, it breaks the useEffect.

Reply:
Makes sense, I had a feeling that might happen. No need to break things for this suggestion!!